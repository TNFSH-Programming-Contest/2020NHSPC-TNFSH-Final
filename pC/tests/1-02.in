5 10 2
3 2 643769
3 5 230959
3 1 846321
4 3 180263
4 1 384812
4 5 562073
1 5 667001
2 5 40457
4 2 988518
2 1 453552
