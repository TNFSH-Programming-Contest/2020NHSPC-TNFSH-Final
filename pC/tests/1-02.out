453552 0 542832 988518 40457 
